doc-Managing_Authentication/topics/External_AD.adoc

```diff
@@ -0,0 +1,158 @@
+[[external_active_directory]]
+= Configuring Authentication with Active Directory
+//source content https://access.redhat.com/solutions/2751431
+
+This procedure outlines how to configure {product-title_short} to authenticate against an existing Active Directory (AD) configuration using external HTTP authentication. This provides Active Directory users access to the {product-title_short} appliance user interface, as well as the REST API.
+
+
+// EDIT A SUMMARY FOR THIS To enable external authentication using Active Directory complete the following steps to configure your HTTP server, then your {product-title_short} appliance.
+
+// Do we need Network Manager step from MIQ docs still?)
+
+. Connect to the {product-title_short} using SSH.
```

Comment:
Add the word - appliance? Connect to the {product-title_short} appliance using SSH.

Reply:
Much better - fixed.